# zrj: add missing link against -lm in libHScrypto-random-*.so
# NOTYPE  GLOBAL DEFAULT  UND log
# fixes security/hs-clientsession (binutils 2.27 ld.gold is unhappy)
--- crypto-random.cabal.orig	2015-03-18 15:42:48.000000000 +0200
+++ crypto-random.cabal
@@ -13,6 +13,7 @@ Homepage:            http://github.com/v
 Cabal-Version:       >=1.6
 
 Library
+  extra-libraries: m
   Build-depends:     base >= 4 && < 5
                    , bytestring
                    , securemem
